src/uct/ib/mlx5/ib_mlx5_dv.c

```diff
@@ -20,6 +20,11 @@ ucs_status_t uct_ib_mlx5dv_init_obj(uct_ib_mlx5dv_t *obj, uint64_t type)
     int ret;
 
     ret = mlx5dv_init_obj(&obj->dv, type);
+#if HAVE_IBV_EXP_DM
```

Comment:
should it be HAVE_IBV_DM? since it uses MLX5DV_OBJ_DM?

Reply:
It's fallback when there are no DM in mlx5dv